screens/disclaimer/DisclaimerScreen.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { StyleSheet, Text, View, Image } from 'react-native';
+import { Spacing, Typography } from '../../theme';
+import Button from '../../components/general/Button';
+import { Asset } from 'expo-asset';
+import useTracking from '../../hooks/useTracking';
+
+const DisclaimerScreen: React.FC = () => {
+  const askPermission = useTracking().askPermission;
+  return (
+    <View style={styles.container}>
+      <Text style={styles.titleText}>Location disclaimer</Text>
+      <Text style={styles.bodyText}>
+        This app collects location data to enable background tracking. Note that your location is never sent anywhere
+        nor shared without your concent.
+      </Text>
+
+      <Text style={styles.bodyText}>
+        The main functionality of the app is to let users to record a track when they are located within a particular
+        geofence. Your activity can be started, paused, stopped at any time. Aditionally we will not send your
+        activity(including your geofence location) before you choose to publish the tracked activity. You can at any
+        time discard the activity.
+      </Text>
```

Comment:
Some spelling errors. New suggestion:

The main functionality of this application is to let users track their position when they are located in pre-defined geofences. An activity can be started, paused, and stopped at any time. We will not send your location data to our servers unless you actively decide to publish and share your activity. You may also, at any time during, choose to discard the activity.